src/styles/main.scss

```diff
@@ -0,0 +1,87 @@
+$stopwatch-size: 500px;
+$hand-width: 10px;
+$small-hand-height: $stopwatch-size / 3;
+$center: $stopwatch-size / 2;
+
+//a minute is arranged for 6s since it is boring to watch
+$animation-minute-time: 6s;
+
+@mixin watchsize {
+  width: $stopwatch-size;
+  height: $stopwatch-size;
+}
+
+body {
+  margin: 0;
+}
+
+.page__container {
+  margin: 50px auto;
+}
+
+.container {
+  position: relative;
+
+  @include watchsize();
+
+  border-radius: 50%;
+
+  background-color: #ededed;
+
+  &__hand {
+    position: absolute;
+    top: $center - $small-hand-height;
+    left: ($stopwatch-size / 2) - ($hand-width / 2);
+
+    animation-name: rotate;
+    animation-delay: 1s;
+    animation-duration: $animation-minute-time * 60;
+    animation-timing-function: linear;
```

Comment:
Maybe you could combine these into one line to save space.

Reply:
done